source/common/http/header_map_impl.cc

```diff
@@ -279,7 +278,7 @@ bool HeaderMapImpl::operator==(const HeaderMapImpl& rhs) const {
 }
 
 void HeaderMapImpl::insertByKey(HeaderString&& key, HeaderString&& value) {
-  StaticLookupEntry::EntryCb cb = static_lookup_table_.find(key.c_str());
+  StaticLookupEntry::EntryCb cb = ConstSingleton<StaticLookupTable>::get().find(key.c_str());
```

Comment:
I'm not sure why there would be one in each TU. There's a single ConstSingleton<T> type for every T that is in the global namespace, so the linker should treat this as shared AFAICT.